packages/web-components/fast-foundation/docs/guide/styling-components.doc.md

```diff
@@ -3,4 +3,91 @@ id: styling-components
+
+### Design System Properties
+The following properties are provided by the `FASTDesignSystemProvider` and should be used as appropriate.
+
+| Name                                   | Type | Description   |
+|----------------------------------------|--------------------  |----------------------------------------------------------------------------|
+| `--background-color`                   | `string` (hex color) | Defines the background color of the node. This is used by color recipes and should represent the color UI is rendering on for a given node tree. |
+| `--density`                            | `number`             | A multiplier to control the density of UI elements.                        |
+| `--design-unit`                        | `number`             | The core sizing unit that all sizes are derived from.                      |
+| `--base-height-multiplier`             | `number`             | The number of designUnits used for component height at the base density.   |
+| `--base-horizontal-spacing-multiplier` | `number`             | The number of designUnits used for horizontal spacing at the base density. |
+| `--corner-radius`                      | `number`             | The corner radius of controls.                                             |
+| `--outline-width`                      | `number`             | The width of the outline of outline controls.                              |
+| `--focus-outline-width`                | `number`             | The width the focus indicator.                                             |
+| `--disabled-opacity`                   | `number`             | Opacity of disabled controls.                                              |
+
+## Conditional Stylesheets
+### Forced-colors Stylesheets
+FAST has a commitment to facilitating accessible web experiences and [forced-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors) support is a core tenant of that commitment. `@microsoft/fast-components` exports the `forcedColorsStylesheetBehavior` utility to provide a simple mechanism to apply forced-color stylesheets without bloating the component stylesheet in non-forced-color environments:
+
+__Example: Forced-colors stylesheets__
+```js
+import { forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+const styles = css`
+    /* ... */
+`.withBehaviors(
+    forcedColorsStylesheetBehavior(css`
+        :host {
+            background: ${SystemColors.Canvas};
+            color: ${SystemColors.CanvasText};
+        }
+        /* ... */
+    `)
+)
+```
+
+### MatchMedia Stylesheets
+`forcedColorsStylesheetBehavior` builds is an implementation of the `matchMediaStylesheetBehaviorFactory, which can be used to create custom implementations of stylesheets that will be conditionally attached based on a [MediaQueryList](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList).
```

Comment:
I think there is an unmatched trailing "`" that's causing the entire statement to continue on with blue font color.